api/client/utils.go

```diff
@@ -260,7 +260,7 @@ func (cli *DockerCli) monitorTtySize(id string, isExec bool) error {
 	sigchan := make(chan os.Signal, 1)
 	gosignal.Notify(sigchan, signal.SIGWINCH)
 	go func() {
-		for _ = range sigchan {
+		for range sigchan {
```

Comment:
This is not valid for Go < 1.4.  Maybe we could keep this the same as gccgo cannot even do the normal stuff let along this.